src/status_im/activity_center/core_test.cljs

```diff
@@ -9,13 +9,105 @@
             [status-im.test-helpers :as h]
             [status-im2.setup.config :as config]))
 
+(def notification-id "0x1")
+
 (defn setup []
   (h/register-helper-events)
   (rf/dispatch [:setup/app-started]))
 
-;;;; Contact verification
+(defn test-log-on-failure
+  [{:keys [before-test notification-id event action]}]
+  (rf-test/run-test-sync
+   (setup)
+   (h/using-log-test-appender
+    (fn [logs]
+      (when before-test
+        (before-test))
+      (h/stub-fx-with-callbacks ::json-rpc/call :on-error (constantly :fake-error))
+
+      (rf/dispatch event)
 
-(def notification-id 24)
+      (is (= {:args  [(str "Failed to " action)
+                      {:notification-id notification-id
+                       :error           :fake-error}]
+              :level :warn}
+             (last @logs)))))))
+
+;;;; Misc
+
+(deftest mark-as-read-test
+  (testing "does nothing if the notification ID cannot be found in the app db"
+    (rf-test/run-test-sync
+     (setup)
+     (let [spy-queue (atom [])]
+       (h/spy-fx spy-queue ::json-rpc/call)
+       (let [notifications {types/one-to-one-chat
+                            {:all    {:cursor "" :data [{:id   notification-id
+                                                         :read false
+                                                         :type types/one-to-one-chat}]}
+                             :unread {:cursor "" :data []}}}]
+         (rf/dispatch [:test/assoc-in [:activity-center]
+                       {:notifications notifications
+                        :filter        {:type   types/one-to-one-chat
+                                        :status :all}}])
+
+         (rf/dispatch [:activity-center.notifications/mark-as-read "0x666"])
```

Comment:
is ""0x666" some particular value? maybe putting it in a named `def` would explain what it is.